### DISCUSSION TITLE
Reset selection to select first entry when inputting more text to the input bar

### DISCUSSION THREAD

**Discussion by isak102** [davatorium/rofi#1750] [Q&A] [answered]
Is there a configuration option to select the first entry when typing letters to the input bar? This is how it works in dmenu and it feels more intuitive to me. The current behaviour is that if you scroll down to for example the 4th entry and then start to type more text in the input bar the 4th entry will still be selected, it would be cool if there was an option to reset it to select the first entry again after I start to type something new

**Comment by DaveDavenport** [comment, marked as answer]
There is no option, but you can reproduce this. (See manpage):

```
   Input change
       When the input of the textbox changes:

              configuration {
                inputchange {
                    action: "kb-row-first";
                }
              }
```

**Reply by isak102** [reply]
Wow that was quick! Thank you so much

**Reply by DaveDavenport** [reply]
Its the example from the manpage.
